config/initializers/0_clowder-config.rb

```diff
@@ -14,7 +17,11 @@
   rbac_url = "#{rbac_config.hostname}:#{rbac_config.port}"
 
   # Prometheus
-  prometheus_exporter_config = config.private_dependency_endpoints.dig('compliance', 'prometheus-exporter')
+  if ENV["APPLICATION_TYPE"] == "compliance-prometheus-exporter"
+    prometheus_exporter_config = { hostname: "localhost", port: config.privatePort }
```

Comment:
This should make the two branches of this conditional return an object that behaves similarly:

```suggestion
    prometheus_exporter_config = OpenStruct.new(hostname: "localhost", port: config.privatePort)
```